Keinen 5% Rand bei der Seite

Woersty opened the pull request.

Woersty committed:
Keinen 5% Rand bei der Seite

mschlenstedt commented:
Was spricht denn dagegen? Finde es optisch eigentlich sehr aufgeräumt und ansprechend. Und den Platz zu 100% ausnutzen sollte man sowieso nicht - hat ja nicht jeder einen Monitor mit 1900 Px Breite...

Woersty committed:
RPi-Monitor-Preparation

RPi-Monitor-Preparation für LoxBerry 0.2.3

Woersty committed:
RPi-Monitor - Autoupudate Package info

Woersty committed:
Grober Fehler beim Initscript :(

mschlenstedt commented:
Wollten wir die Monitorgeschichte nicht als Plugin ausführen? Ich möchte das Basissystem so schlank wie möglich halten.

Auch komme ich jetzt nicht mehr klar war hier zu welchem Projekt gehört? Brauchst Du die 0% Rand jetzt für das monitor-Plugin? Oder willst Du das generell (wenn ja: Warum?).

Woersty commented:
Generell, um soviel Platz auf der Seite wie möglich zu haben.
Wollte die anderen Templates dann ggf anpassen.
Aber momentan für das Monitor-Plugin, ja.

Zum Thema schlank, ich dachte man könnte den RPi Monitor so einbinden um keinen Entwicklungsaufwand zu haben und immer von der neuesten Version zu profitieren.
Systemleistung hat der bei mir nicht merklich viel gefressen.

> Am 12.09.2016 um 08:17 schrieb mschlenstedt notifications@github.com:
> 
> Wollten wir die Monitorgeschichte nicht als Plugin ausführen? Ich möchte das Basissystem so schlank wie möglich halten.
> 
> Auch komme ich jetzt nicht mehr klar war hier zu welchem Projekt gehört? Brauchst Du die 0% Rand jetzt für das monitor-Plugin? Oder willst Du das generell (wenn ja: Warum?).
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

mschlenstedt commented:
Ok. Ich wprde die 5% gerne lassen - weil optisch einfach ansehnlicher.

Zum RPi-Monitor: Also bauen wir es in das System ein? Kannst Du dann Deine Änderungen bitte noch einmal als Pull Request gegen 0.3.0 machen. Ich möchte den aktuellen 0.2.2 Branch nur für Bugfixes nutzen - ohne neue Features.

ich erstelle gleich noch den 0.3.0 Branch.

Woersty committed:
Europa nach oben

Europa zusätzlich an den Anfang der Liste und die drei deutschsprachigen
Hauptstädte nach oben sortiert.
Fixes #123

Woersty commented:
Bitte 4% - dann fummel ich das nochmal um.

Aber 4% ist Minimum bei 1920x1080

Mit freundlichem Gruß
Christian Wörstenfeld

---

Christian Wörstenfeld
Kalkbergeweg 10
D-12589 Berlin
Telefon: +49 30 50 15 8484
Mailbox: +49 30 55 15 1230
Fax:     +49 30 55 15 1232
Handy:   +49 30 55 15 1234
e-mail:christian@woerstenfeld.de
Threema: SV3C7MUX
Web: www.woerstenfeld.de
¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯

Am 12.09.2016 um 21:22 schrieb mschlenstedt:

> Ok. Ich wprde die 5% gerne lassen - weil optisch einfach ansehnlicher.
> 
> Zum RPi-Monitor: Also bauen wir es in das System ein? Kannst Du dann 
> Deine Änderungen bitte noch einmal als Pull Request gegen 0.3.0 
> machen. Ich möchte den aktuellen 0.2.2 Branch nur für Bugfixes nutzen 
> - ohne neue Features.
> 
> ich erstelle gleich noch den 0.3.0 Branch.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> https://github.com/mschlenstedt/Loxberry/pull/122#issuecomment-246461315, 
> or mute the thread 
> https://github.com/notifications/unsubscribe-auth/ATga5PN5xQBDgl3MNhnesc80O9WfyixUks5qpaZ7gaJpZM4J6EfH.

Woersty committed:
4% Rand

Woersty committed:
4% Rand
